packages/binutils.rb

```diff
@@ -4,44 +4,56 @@ class Binutils < Package
   description 'The GNU Binutils are a collection of binary tools.'
   homepage 'https://www.gnu.org/software/binutils/'
   @_ver = '2.36.1'
-  version @_ver
+  version "#{@_ver}-1"
   license 'GPL-3+'
   compatibility 'all'
   source_url "https://ftpmirror.gnu.org/binutils/binutils-#{@_ver}.tar.xz"
   source_sha256 'e81d9edf373f193af428a0f256674aea62a9d74dfe93f65192d4eae030b0f3b0'
 
   binary_url({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/binutils-2.36.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/binutils-2.36.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/binutils-2.36.1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/binutils-2.36.1-chromeos-x86_64.tar.xz'
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/binutils-2.36.1-1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/binutils-2.36.1-1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/binutils-2.36.1-1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/binutils-2.36.1-1-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: '22a88c3eb29672805441c78fa2c37414a4d49bec790252391c1caf45edc20426',
-     armv7l: '22a88c3eb29672805441c78fa2c37414a4d49bec790252391c1caf45edc20426',
-       i686: '7f47210024759db0485292d7695f46532ae0c4c579c03ba0c6450a5174ce9f6f',
-     x86_64: '624fb52ce022d6caea7c99a8c6a08f872e51fe0c8be09bb3c3f72e186ad60d3a'
+    aarch64: '3f580f50aa890e57b2faead8c500982890b03c834a0808d1ee8fbf8f91df706f',
+     armv7l: '3f580f50aa890e57b2faead8c500982890b03c834a0808d1ee8fbf8f91df706f',
+       i686: '791e202c0fa73426b35ba072f4bd7d08150792d56ed0a55fd92921373eeef1ed',
+     x86_64: '564af95307c729f0199717dbb770a7dae08c3cb3818298689f809076c45fa9c5'
   })
 
-  def self.build
+  def self.patch
     system 'filefix'
+    system "sed -i 's,scriptdir = \$(tooldir)/lib,scriptdir = \$(tooldir)/#{ARCH_LIB},g' ld/Makefile.am"
+    Dir.chdir 'ld' do
+      system 'aclocal && automake'
+    end
+  end
+
+  def self.build
     Dir.mkdir 'build'
     Dir.chdir 'build' do
       system "env CFLAGS='-pipe -flto=auto' CXXFLAGS='-pipe -flto=auto' \
         LDFLAGS='-flto=auto' \
         ../configure #{CREW_OPTIONS} \
-        --with-system-zlib \
+        --disable-bootstrap \
         --disable-maintainer-mode \
-        --enable-shared \
+        --enable-64-bit-bfd \
```

Comment:
Only applies to x86_64?

Reply:
yeah, according to binutils's gentoo ebuild, --enable-64-bit-bfd is only relevant on 64 bit processors and is much slower than the 32 bit bfd so should be omitted on 32 bit system. Maybe add this and use the variable appropriately.
```
case ARCH
when 'i686' or 'armv7l'
  @_64_bit_bfd = '--disable-64-bit-bfd'
when 'x86_64' or 'aarch64'
  @_64_bit_bfd = '--enable-64-bit-bfd'
end
```

Reply:
You will need to disable for aarch64 also since the userland is only armv7l compatible:

Reply:
The reason I left it in was that this is apparently used on non-64-bit architectures too:

https://lfs.maru-na.net/lfs/view/systemd/chapter08/binutils.html

```
--enable-64-bit-bfd
Enables 64-bit support (on hosts with narrower word sizes). May not be needed on 64-bit systems, but does no harm.
```

(Apparently it is required on 32-bit systems if you want to build some things like grub. Src: https://schlomp.space/teldra/vp-build/commit/1dcd80236d7032da541efa1b1d8a687ab6fbc0d3 )

Reply:
The relevant gentoo bug report is from 9 years ago: https://bugs.gentoo.org/438522 

Is this slowness from memory pressure still relevant? I'm compiling i686 and armv7l packages in containers with > 4Gb of ram, so this shouldn't even be an issue.

Reply:
Debian doesn't use it at all, so maybe I should just remove it?

https://salsa.debian.org/toolchain-team/binutils/-/blob/master/debian/rules

Reply:
I'm going to try to avoid arch specific customizations here, so I'll eliminate that line. If it is done automatically for the relevant architectures, great!